Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9 AS base
```

Comment:
https://github.com/RedHatInsights/insights-rbac/pull/782
seems we are using minimal on purpose for the Fedramp: https://issues.redhat.com/browse/RHCLOUD-23571
We might want to keep the minimal? @coderbydesign @lpichler

Reply:
I also know that while we _can_ support ubi9 in that environment now, we would likely need to prevent the base image from picking up a new version before it's approved. Not sure if/how that impacts the needs here?

Reply:
The minimal base image within the dockerfile now seems to not have the subscription-manager-rhsm-certificates package, so when the container is being built within konflux, it returns an issue with the SSL CA Certs trying to validate the connection but it can't find the certs. 
From what I'm understanding your comment, we need to wait for the ubi9 update, so maybe I can create a separate Dockerfile just for konflux at the moment to use the non-minimal ubi8 like in the changes I made within the Dockerfile?

Reply:
Update: going to revert the PR back to its original state and then get all the konflux updates in so that we can create a new branch for troubleshooting the issue and have a good baseline for trouble shooting